### PR TITLE
Made event post for DDOW

### PR DESCRIPTION
**Pull request by fat1996**

### CODE REVIEW

File: content/W2017/W17-blog-DDOW.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
++title: Distributed Denial of Women
+ +Date: 2017-02-24
+ +Category: Blog
+ +Tags: feminism, news
+ +Slug: Distributed-Denial-Of-Women-W17
+ +Author: Fatima Taj
+ +Summary: WiCS thanks everyone who participated in the Distributed Denial of Women on Feb 23rd, 2017!
+ +
+ +Women play a crucial role in the tech industry but are often 
+ +overlooked and undervalued. WiCS thanks everyone who participated in
```
Comment (annalorimer) [inline content/W2017/W17-blog-DDOW.md]: WiCS thanks -> WiCS stands with

File: content/W2017/W17-blog-DDOW.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
++title: Distributed Denial of Women
+ +Date: 2017-02-24
+ +Category: Blog
+ +Tags: feminism, news
+ +Slug: Distributed-Denial-Of-Women-W17
+ +Author: Fatima Taj
+ +Summary: WiCS thanks everyone who participated in the Distributed Denial of Women on Feb 23rd, 2017!
```
Comment (annalorimer) [inline content/W2017/W17-blog-DDOW.md]: WiCS thanks -> WiCS stands with

Remove the exclamation points too.

File: content/W2017/W17-blog-DDOW.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+ +
+ +Women play a crucial role in the tech industry but are often 
+ +overlooked and undervalued. WiCS thanks everyone who participated in 
+ +the [Distributed Denial of Women](http://distributed-denial-of-women.org/) on February 23rd, 2017
```
Comment (annalorimer) [inline content/W2017/W17-blog-DDOW.md]: lines need to be 80 char 😊

File: content/W2017/W17-blog-DDOW.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ +Category: Blog
+ +Tags: feminism, news
+ +Slug: Distributed-Denial-Of-Women-W17
+ +Author: Fatima Taj
+ +Summary: WiCS thanks everyone who participated in the Distributed Denial of Women on Feb 23rd, 2017!
+ +
+ +Women play a crucial role in the tech industry but are often 
+ +overlooked and undervalued. WiCS thanks everyone who participated in 
+ +the [Distributed Denial of Women](http://distributed-denial-of-women.org/) on February 23rd, 2017 
+ +and stood up for women and non-binary people, who are essential to tech.
```
Comment (annalorimer) [inline content/W2017/W17-blog-DDOW.md]: women -> cis and trans women

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Comment by annalorimer** [comment]
@fat1996 ping!

**Comment by fat1996** [comment]
@annalorimer should I replace all mentions of women with cis and trans women? I wasn't sure which one you were referring to.

**Comment by annalorimer** [comment]
@fat1996 good question. @evykassirer might have a better way of saying something like 'gender minorities' but better worded.

**Comment by fat1996** [comment]
@evykassirer could you give me some suggestions as to how to reword it? replace 'women' with 'cis and trans women, non binary individuals and other gender minorities'?

**Comment by evykassirer** [comment]
hmm I think "gender minorities" is concise but not very specific (we generally don't include trans men, for example)

I think we should stick with what we always say and do "cis and trans women, and non-binary individuals"

**Comment by bdamerac** [comment]
There's still the exclamation. Otherwise, looks good to me.

**Comment by annalorimer** [comment]
There shouldn't be any '+' before the fields at the top of the file. I'll remove these and merge.

**Pull request merged**